todo-src/script.js

```diff
@@ -5,12 +5,14 @@ var myApp = angular.module('app', []);
 myApp.controller('MainCtrl', function ($scope){
```

Comment:
I changed this to an array of objects instead of strings. Notice how on my version looks like something you'd see in a mongo db.